.github/workflows/node.js.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Test
+on:
+  push:
+    branches: [ master, f/*, b/* ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x]
```

Comment:
I think, including node 10x it's a good ideia since several services (like firebase) still not support node 12x

Reply:
I've added it. I wanted to use `Array.flat()` but the reduce version is also a oneliner.